src/components/SelectAccount.tsx

```diff
@@ -11,16 +13,30 @@ const SelectAccount = () => {
     s.multisigAccounts,
   ]);
 
+  const listMultiCliqueAccounts = usePromise({
```

Comment:
Not sure if I'm integrating `listMultiCliqueAccounts` in correct page as this page displays `Multisig[]` while `listMultiCliqueAccounts` returns `MultiCliqueAccounts[]`